packages/jazz-tools/src/tools/testing.ts

```diff
@@ -392,3 +395,11 @@ export function disableJazzTestSync() {
   }
   syncServer.current = null;
 }
+
+export function assertLoaded<T extends MaybeLoaded<CoValue>>(
```

Comment:
I think it would be useful to have this utility in `jazz-tools`

I would use this in some functions where I expect the value to be loaded successfully.